schema/interface.acl.yml

```diff
@@ -0,0 +1,39 @@
+description: A collection of access control entries that define the rules for filtering traffic through a network interface.
+type: array
+items:
+  type: object
+  properties:
+    acl-inf-policy-preference:
+      description: Determines the priority of multiple ACL policies when more than one is applied to an interface, if any.
+      type: integer
+      minimum: 1
+      maximum: 64
+      default: 1
+    acl-inf-policy-ingress:
+      description: Specifies the ACL policy that is applied to incoming traffic on an interface.
+      type: string
```

Comment:
Do we want to have maybe a list of enums for string? Either way it seems like we could be having per-platform stuff supported here?
E.g. each OLS vendor would have it's own definitions;
I'm not sure whether we can have complete listed enum variants here as of now though;

Reply:
Ideally the vendor variants could be rolled up into something common, leaving string makes configuration a bit hit and miss. 
Ideally you have a configuration that can be used on multiple vendor's switches, and the switches translate the schema configuration into their internal model. Otherwise this could be rather different per vendor.

Reply:
@Cahb @mikehansen1970, Sometimes ACL policies in ports are specified by integer numbers, and these integers map to policies that are created without names as well... So the key question here may be ahow do we specifying the ACL policy to apply on a port. 
An array of 'policies' is what this schema applies on a given port.
And each policy i thought can be abstracted to a string, which could be a Key value that could be a integer or a name or some reference.. which may be anything.